webui/module/Client/view/client/client/index.phtml

```diff
@@ -111,6 +111,53 @@ $this->headTitle($title);
       }
    }
 
+   function formatUname(value) {
+      if(value.toLowerCase().search("suse") > -1) {
+         return '<img src="<?php echo $this->basePath('img/icons/os/suse.png'); ?>" id="icon-os" title="'+value+'" data-toggle="tooltip" data-placement="top">';
```

Comment:
Could split this into another helper function the re-uses the redundant text?

Reply:
Of course, more refactoring of a few JS functions will follow after another iteration.